posthog/api/action.py

```diff
@@ -363,6 +365,13 @@ def trends(self, request: request.Request, *args: Any, **kwargs: Any) -> Respons
             # If no filters, automatically grab all actions and show those instead
             filter.entities = [Entity({'id': action.id, 'name': action.name, 'type': TREND_FILTER_TYPE_ACTIONS}) for action in actions]
 
+        if not filter.date_from:
+            filter._date_from = Event.objects.filter(team=team)\
+                .order_by('timestamp')[0]\
+                .timestamp\
+                .replace(hour=0, minute=0, second=0, microsecond=0)\
+                .isoformat()
+
```

Comment:
Handle the case where `if not filter.date_to` here and setting `filter._date_to=now().isoformat()`